src/java/org/apache/cassandra/schema/DistributedSchema.java

```diff
@@ -30,7 +30,7 @@
 {
     public static final DistributedSchema EMPTY = new DistributedSchema(Keyspaces.none(), SchemaConstants.emptyVersion);
 
-    private final Keyspaces keyspaces;
+    protected final Keyspaces keyspaces;
```

Comment:
NIT: I don't have a strong opinion, but I _think_ that we are using getKeyspaces() in VersionedSchema (which extends this class)

Reply:
Ah yes, I missed that. I see a new use of `keyspaces` came in with https://github.com/riptano/cndb/commit/8d9c63f377adadbf6a85fa3c55e0047bcd6af28d.

OK let's keep it consistent and use `getKeyspaces()` everywhere and I'll drop this part.